packages/js/src/generators/library/library.ts

```diff
@@ -249,6 +255,25 @@ export async function addLint(
   });
 }
 
+function addBundlerDependencies(tree: Tree, options: NormalizedSchema) {
+  updateJson(tree, `${options.projectRoot}/package.json`, (json) => {
+    if (options.bundler === 'tsc') {
+      json.dependencies = {
+        ...json.dependencies,
+        tslib: tsLibVersion,
+      };
+    } else if (options.bundler === 'swc') {
+      json.dependencies = {
+        ...json.dependencies,
+        '@swc-node/register': swcNodeVersion,
+        '@swc/core': swcCoreVersion,
```

Comment:
I thought `@swc/helpers` was the only one needed? It's the only one we add when we generate the package.json. Why are these necessary here?

Reply:
Removed, that changed after @jaysoo's PR